games/ninswitch/game_imagerec.py

```diff
@@ -69,6 +89,12 @@ async def on_init(self):
         self.image_rec_task = asyncio.create_task(self.image_rec_main())
         self.image_rec_task.add_done_callback(self.image_rec_done_cb)
 
+        self.io.register_inputs(
+            {
+                "capture_frame": capture_screen(self.cap),
```

Comment:
Move this with the rest of the registered inputs above

Reply:
In my testing, I found that having it before the other image_rec_task that were there normally caused issues. I will look into it again and see if I can get that working.